DataFormats/TauReco/interface/RecoTauPiZero.h

```diff
@@ -14,32 +14,57 @@ class RecoTauPiZero : public CompositePtrCandidate {
       kStrips = 3
     };
 
-    RecoTauPiZero():CompositePtrCandidate(),algoName_(kUndefined){
-      this->setPdgId(111); }
-
-    RecoTauPiZero(PiZeroAlgorithm algoName):
-        CompositePtrCandidate(), algoName_(algoName) { this->setPdgId(111); }
+    RecoTauPiZero()
+      : CompositePtrCandidate(),
+        algoName_(kUndefined)
```

Comment:
@slava77 I'm not sure what you mean here - does this mean set bendCorrEta_ = 0 (e.g.) outside of the RecoTauPiZero() : CompositePtrCandidate(), algoName_ brackets?